src/server.cpp

```diff
@@ -38,6 +75,8 @@ void Server::handleReceive(const error_code& error, std::size_t bytes)
     {
         Packet packet = makeResponse(Packet(m_recvBuffer, bytes));
 
+        printPacket(packet);
```

Comment:
Let's mention that this is a response packet. Just a `std::cout` before the call.